arangod/Cluster/AgencyCache.cpp

```diff
@@ -1,8 +1,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 /// DISCLAIMER
 ///
-/// Copyright 2014-2020 ArangoDB GmbH, Cologne, Germany
-/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+/// Copyright 2020 ArangoDB Inc, Cologne, Germany
```

Comment:
It seems we are just copying entire files over from 3.7, which is probably not good.